### ISSUE TITLE
Websocket fuzzer not using script generated payloads

### ISSUE THREAD

**Issue by blueigloo** [zaproxy/zaproxy#3017]
I am testing an application that is using websockets.  I wrote a payload generator in python based off of the provided bruteforce.py.

To reproduce:
1. Select message payload from the websocket tab
2. Highlight what I want to fuzz and then right-click and select Fuzz...
3. Click the payload button and add the script that I've added via the script editor.
4. Select okay a few times and then start the fuzzer.
In the fuzzing pane it looks like it is using the fuzzing data, however when I check the websocket channel the same original message is just being resent over and over again.

**Comment by thc202** [comment]
Are you using the latest versions of the add-ons?

Could you provide the (minimal) script that reproduces the issue? Just tried the steps mentioned and it fuzzed as expected (messages fuzzed and shown in the WebSockets tab were the same and matched the ones received by the target endpoint).

**Comment by thc202** [comment]
@blueigloo any more details that you could provide?

**Comment by thc202** [comment]
@blueigloo any other info that you could share?

**Comment by kingthorin** [comment]
User created an account to open this issue but doesn't have any other activity in 9mo.

I propose this be closed as a won't fix since we don't have enough detail to re-create.

**Comment by thc202** [comment]
Agreed. Closing, insufficient evidence. If anyone happens to reproduce the issue, please, let us know.

**Comment by thc202** [comment]
Reopening, this happens with binary payloads, they are currently not fuzzed. We should disallow fuzzing binary payloads (since it's not working) or just implement that.

**Comment by thc202** [comment]
BTW, the title was updated to reflect the actual issue.
